samples/snippets/quickstart_sample.py

```diff
@@ -17,73 +17,49 @@
+):
     # You must set the api_endpoint if you use a location other than 'us', e.g.:
-    opts = {}
-    if location == "eu":
-        opts = {"api_endpoint": "eu-documentai.googleapis.com"}
+    opts = ClientOptions(api_endpoint=f"{location}-documentai.googleapis.com")
 
     client = documentai.DocumentProcessorServiceClient(client_options=opts)
 
     # The full resource name of the processor, e.g.:
-    # projects/project-id/locations/location/processor/processor-id
+    # projects/project_id/locations/location/processor/processor_id
     # You must create new processors in the Cloud Console first
-    name = f"projects/{project_id}/locations/{location}/processors/{processor_id}"
+    name = client.processor_path(project_id, location, processor_id)
 
     # Read the file into memory
     with open(file_path, "rb") as image:
         image_content = image.read()
 
-    document = {"content": image_content, "mime_type": "application/pdf"}
+    # Load Binary Data into Document AI RawDocument Object
+    raw_document = documentai.RawDocument(content=image_content, mime_type=mime_type)
 
     # Configure the process request
-    request = {"name": name, "raw_document": document}
+    request = documentai.ProcessRequest(name=name, raw_document=raw_document)
 
     result = client.process_document(request=request)
-    document = result.document
-
-    document_pages = document.pages
 
-    # For a full list of Document object attributes, please reference this page: https://googleapis.dev/python/documentai/latest/_modules/google/cloud/documentai_v1beta3/types/document.html#Document
+    # For a full list of Document object attributes, please reference this page:
+    # https://cloud.google.com/python/docs/reference/documentai/latest/google.cloud.documentai_v1.types.Document
+    document = result.document
 
     # Read the text recognition output from the processor
-    print("The document contains the following paragraphs:")
-    for page in document_pages:
-        paragraphs = page.paragraphs
-        for paragraph in paragraphs:
-            print(paragraph)
-            paragraph_text = get_text(paragraph.layout, document)
-            print(f"Paragraph text: {paragraph_text}")
-
-
-def get_text(doc_element: dict, document: dict):
-    """
-    Document AI identifies form fields by their offsets
-    in document text. This function converts offsets
-    to text snippets.
-    """
-    response = ""
-    # If a text segment spans several lines, it will
-    # be stored in different text segments.
-    for segment in doc_element.text_anchor.text_segments:
-        start_index = (
-            int(segment.start_index)
-            if segment in doc_element.text_anchor.text_segments
-            else 0
-        )
-        end_index = int(segment.end_index)
-        response += document.text[start_index:end_index]
-    return response
+    print("The document contains the following text:")
+    print(document.text)
```

Comment:
Same question about document.txt vs get_text

Reply:
This example is exactly the same as the online processing example. Focused on making the processing request, not parsing the response. The OCR/Form parsing/etc examples are focused on handling the response, not constructing the request